src/store/modules/recipeStore.js

```diff
@@ -107,5 +121,44 @@ export const recipeStore = {
         alert("There was an error creating the recipe");
       }
     },
+
+    async update_recipe(context, payload) {
+      const uri =
+      process.env.BASE_URL + "recipes" +
+        "/" +
+        payload.recipeId +
+        "/edit";
+      try {
```

Comment:
axios provides a `.then()` and a `.catch()` function, as success and failure handlers respectively. It's better to use those instead of using a try catch block. Your approach introduces a redendancy, where you are checking the response object for a status code and you also have a catch block. I advice to use the function mentioned above instead and in the .catch function call handle_error context.

Reply:
has been changed